lib/project_types/node/commands/create.rb

```diff
@@ -26,7 +26,11 @@ def call(args, _name)
         check_npm
         build(form.name)
 
-        ShopifyCli::Project.write(@ctx, 'node')
+        ShopifyCli::Project.write(
+          @ctx,
+          app_type: 'node',
```

Comment:
should we be opinionated about whether or not it gets written as a symbol or a string, since we are opinionated about writing IDs as ints and not strings?

Reply:
I do not think it matters, I think the main difference is that it will be serialized differently and the yaml will look different so it is superficial because the functionality that retrieves it will always return a symbol.